src/read.c

```diff
@@ -3440,13 +3440,19 @@ static avifResult avifParseTrackBox(avifDecoderData * data,
     AVIF_CHECKERR(track != NULL, AVIF_RESULT_OUT_OF_MEMORY);
 
     avifBool edtsBoxSeen = AVIF_FALSE;
+    avifBool tkhdSeen = AVIF_FALSE;
     while (avifROStreamHasBytesLeft(&s, 1)) {
         avifBoxHeader header;
         AVIF_CHECKERR(avifROStreamReadBoxHeader(&s, &header), AVIF_RESULT_BMFF_PARSE_FAILED);
 
         if (!memcmp(header.type, "tkhd", 4)) {
+            if (tkhdSeen) {
+                avifDiagnosticsPrintf(data->diag, "[trak] box contains multiple [tkhd] boxes");
```

Comment:
Nit: This error message and the error message at line 3465 (for the "edts" box) should have the same structure.

We may want to use the structure of the error message in `uniqueBoxSeen()`:

"Box[trak] contains a duplicate unique box of type 'tkhd'"

Reply:
Done, updated all the error messages.